lib/SILGen/SILGenType.cpp

```diff
@@ -554,11 +554,6 @@ class SILGenConformance : public SILGenWitnessTable<SILGenConformance> {
     PrettyStackTraceConformance trace("generating SIL witness table",
                                       Conformance);
 
-    // Check whether the conformance is valid first.
-    Conformance->resolveValueWitnesses();
```

Comment:
It still makes sense to call resolveValueWitnesses() I think, and it would be less risk to leave it in

Reply:
The call to `resolveValueWitnesses()` is something I added recently. It feels redundant to me now that we’re not querying any result of it and witness table emission itself will also trigger resolution if necessary.

Reply:
Left it in because it is more efficient.